.gitignore

```diff
@@ -1,8 +1,16 @@
 .idea/
 **/.DS_Store
+<<<<<<< HEAD
```

Comment:
```
<<<<<<< HEAD
=======
>>>>>>> master
```
these lines should be removed from the file; also, there are duplicated definitions in the file